elasticsearch/install.sh

```diff
@@ -14,7 +14,7 @@ export CLUSTER_NAME=placeholder
 
 mkdir -p ${HOME}
 ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
-/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5
+/usr/share/elasticsearch/bin/plugin -i search-guard -url https://github.com/lukas-vlcek/origin-aggregated-logging/releases/download/v0.1/search-guard-0.5.1.zip
```

Comment:
Can you use the following plugin name to patch the previous formatting? `com.floragunn/search-guard/0.5.1`